index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SciFacts homepage</title>
+    <link rel="stylesheet" href="./css/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,300&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+    <nav>
+    <div id="topbar">
+        <ul class="icons">
+            <li>
+                <a href="www.twiiter.com/mike-dev"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li> 
+                <a href="www.linkedin/mike-dev"><i class="fa fa-linkedin"></i></a>
+            </li>
+        </ul> 
+        <a href="#"><p>English</p></a>
+        <a href="#"><p>My page</p></a>
+    </div>
+    </nav>
+
+    <nav id="navbar">
+        <a href="index.html"><div id="navbar_logo"></div></a>
+        <ul id="nav_liststyle">
+            <li><a href="index.html" class="menu_list home_color">Home</a></li>
+            <li><a href="about.html" class="menu_list">About</a></li>
+        </ul>
+    </nav>
+
+    <header id="header">
+        <div id="large_screen">
+            <div class="hero_content">
+                <nav id="menu_icon">
+                    <a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a>
+                </nav> 
+                <p class="hello_text">"Hello! great learners"</p>
+                <h1>Science Fun Facts</h1>
+                <p class="brief">The history of science is almost as old as time itself. Science is constantly working 
+                    from the edges of the universe to the innermost cells that make you function
+                </p>
+                <p class="date">All about science</p>
+                <p class="location">Its Latin word, <i>scienta</i> means knowledge.</p>
+            </div> 
+        </div>    
+    </header>
+
+    <div id="menu">
+
+        <a href="" id="close_icon"><i class="fa fa-remove"></i> </a>
+            <ul id="menu_liststyle">
+                <li><a href="index.html" class="menu_list home_color">Home</a></li>
+                <li><a href="about.html" class="menu_list">About</a></li>
+            </ul>
+
+    </div>
+
+    <section class="sci_facts">
+       <div class="title">
+        <h2 class="h2_sci_facts">Some Science facts</h2>
+        <div class="line"></div>
+       </div>
+
+       <div id="desk_facts">
```

Comment:
- [ ] These articles are five and not six kindly fix this, also they have to be wrapped in an article tag and not the div tag

#### Your design
![image](https://user-images.githubusercontent.com/81932478/190414931-16c9bc72-999e-4d96-af7c-5e08fd4b0fab.png)

#### Given design
![image](https://user-images.githubusercontent.com/81932478/190415075-e0933611-0ceb-449b-bd67-ec9808d5e382.png)